Terminal.Gui/Core/View.cs

```diff
@@ -2501,12 +2608,108 @@ bool SetWidthHeight (Rect nBounds)
 			}
 			if (aSize) {
 				Bounds = new Rect (Bounds.X, Bounds.Y, canSizeW ? rW : Bounds.Width, canSizeH ? rH : Bounds.Height);
-				TextFormatter.Size = Bounds.Size;
+				TextFormatter.Size = GetBoundsTextFormatterSize ();
 			}
 
 			return aSize;
 		}
 
+		/// <summary>
+		/// Calculates the size to fit all text if <see cref="AutoSize"/> is true.
```

Comment:
This function returns a `Size`. I think it should be named `GetAutoSize`.

Why are all 3 functions needed? `CalculateAutoSize`, `CalcuulateAutoSizeWidth`, `CalculateAutoSizeHeight`?

Why not just `Size`?

Reply:
> This function returns a `Size`. I think it should be named `GetAutoSize`.

Not all the code that returns something has to have the prefix `Get`, but I agree with the suggestion and I'll rename it.



> Why are all 3 functions needed? `CalculateAutoSize`, `CalcuulateAutoSizeWidth`, `CalculateAutoSizeHeight`?
> 
> Why not just `Size`?

You are right, I forgot to delete the other two. Thanks.